dasgo/resources.go

```diff
@@ -166,36 +175,42 @@ const (
 )
 
 // Select Menu Structure
+
+// *
```

Comment:
Comment needs clarification.

Reply:
same as above type was not in go file

Reply:
These are component types so it's presented with the flags and interface function. Don't include that field.